src/test/java/net/eventsexpress/app/tests/TestSetUp.java

```diff
@@ -0,0 +1,38 @@
+package net.eventsexpress.app.tests;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import io.github.cdimascio.dotenv.Dotenv;
+import net.eventsexpress.app.config.ConfigurationManager;
+import net.eventsexpress.app.driver.DriverManager;
+import net.eventsexpress.app.utils.Utils;
+
+public class TestSetUp {
+    public Dotenv dotenv;
+    public String EMAIL;
+    public String PASSWORD;
+    public String NEW_EMAIL;
+    public String NEW_PASSWORD;
```

Comment:
If this is constants then 'final' and 'static' is expected here.

Reply:
Yes, they won't be changed during the test.